src/AcceptanceTests/ConfigureEndpointAzureServiceBusTransport.cs

```diff
@@ -33,7 +39,39 @@ public Task Configure(string endpointName, EndpointConfiguration config, RunSett
         {
             transportConfig.UseTopology<ForwardingTopology>();
         }
-        //else default
+        else
+        {
+            transportConfig.UseTopology<EndpointOrientedTopology>()
```

Comment:
@melkio while this achieves the goal, I feel that hiding this information from the tests is not ideal... Can I suggest that each ATT tests that does pub/sub contain it's conditional mapping?